README.md

```diff
@@ -41,7 +41,7 @@ sh <(curl https://cyberpanel.net/install.sh || wget -O - https://cyberpanel.net/
 
 
 ```
-sh <(curl https://raw.githubusercontent.com/usmannasir/cyberpanel/stable/preUpgrade.sh || wget -O - https://raw.githubusercontent.com/usmannasir/cyberpanel/stable/preUpgrade.sh)
+sh <(curl https://raw.githubusercontent.com/NNetworkBR/cyberpanel/stable/preUpgrade.sh || wget -O - https://raw.githubusercontent.com/NNetworkBR/cyberpanel/stable/preUpgrade.sh)
```

Comment:
Need to change all these refrences to official repo before i can go through this.

Reply:
> Need to change all these refrences to official repo before i can go through this.

Reverted commits!